src/components/nft-details/nft-details.tsx

```diff
@@ -164,22 +164,31 @@ export const NftDetails = () => {
                 <TraitsListLoader />
               ) : (
                 <>
-                  {Object.keys(nftDetails.traits).map((key) => (
-                    <NFTTraitsChip
-                      label={getTraitName(key)}
-                      key={key}
-                      name={nftDetails?.traits[`${key}`].name}
-                      rimValue={`${
-                        nftDetails?.traits[`${key}`].occurance
-                      } (${
-                        nftDetails?.traits[`${key}`].rarity &&
-                        roundOffDecimalValue(
+                  {Object.keys(nftDetails.traits).map((key) => {
+                    const occurance = nftDetails?.traits[`${key}`]
+                      .occurance
+                      ? nftDetails?.traits[`${key}`].occurance
+                      : undefined;

```

Comment:
The following will fallback to undefined, if you prefer:

```suggestion
                    const occurance = nftDetails?.traits?.[`${key}`]?
                      .occurance;
```